vivisect/parsers/pe.py

```diff
@@ -92,6 +93,7 @@ def loadPeIntoWorkspace(vw, pe, filename=None, baseaddr=None):
     vw.setMeta('Architecture', arch)
     vw.setMeta('Format', 'pe')
     vw.parsedbin = pe
+    vw.setMeta('FileBytes', base64.urlsafe_b64encode(pe.getFileBytes()))
```

Comment:
I don't think we've encountered the need to access the file bytes from within the workspace in our use-cases. If stored I'd recommend to compress the data.

Reply:
I've encountered it enough where I'm tired of keeping around that particular hacky patch file and figured I should turn that into a realboy PR.

But compressing them is a good idea. I'll throw that in.